examples/guestbook/README.md

```diff
@@ -199,81 +276,51 @@ If you want to see the logs for a given container, you can run:
-
-Services find the pods to load balance based on the pods' labels.
-The pod that you created in [Step One](#step-one-start-up-the-redis-master) has the label `name=redis-master`.
-The selector field of the service description determines which pods will receive the traffic sent to the service, and the `port` and `targetPort` information defines what port the service proxy will run at.
+Just like the master, we want to have a service to proxy connections to the redis slaves. In this case, in addition to discovery, the slave service will provide transparent load balancing to web app clients.
 
-The file `examples/guestbook/redis-master-service.yaml` defines the redis master service:
+The service specification for the slaves is in `examples/guestbook/redis-slave-service.yaml`:
 
-<!-- BEGIN MUNGE: EXAMPLE redis-master-service.yaml -->
+<!-- BEGIN MUNGE: EXAMPLE redis-slave-service.yaml -->
 
 ```yaml
 apiVersion: v1
 kind: Service
 metadata:
-  name: redis-master
+  name: redis-slave
   labels:
-    name: redis-master
+    app: redis
+    tier: backend
+    role: slave
 spec:
   ports:
     # the port that this service should serve on
   - port: 6379
-    targetPort: 6379
   selector:
-    name: redis-master
+    app: redis
+    tier: backend
+    role: slave
 ```
 
-[Download example](redis-master-service.yaml?raw=true)
-<!-- END MUNGE: EXAMPLE redis-master-service.yaml -->
-
-Create the service by running:
+[Download example](redis-slave-service.yaml?raw=true)
+<!-- END MUNGE: EXAMPLE redis-slave-service.yaml -->
 
-```console
-$ kubectl create -f examples/guestbook/redis-master-service.yaml
-services/redis-master
-```
+This time the selector for the service is `app=redis,role=slave,tier=backend`, because that identifies the pods running redis slaves. It may also be helpful to set labels on your service itself as we've done here to make it easy to locate them with the `kubectl get services -l "label=value"` command.
 
-Then check the list of services, which should include the redis-master:
+Now that you have created the service specification, create it in your cluster by running:
 
 ```console
+$ kubectl create -f examples/guestbook/redis-slave-service.yaml
+service "redis-slave" created
+
 $ kubectl get services
-NAME              CLUSTER_IP       EXTERNAL_IP       PORT(S)       SELECTOR               AGE
-redis-master      10.0.136.3       <none>            6379/TCP      app=redis,role=master  1h
-...
+NAME             CLUSTER_IP       EXTERNAL_IP       PORT(S)       SELECTOR                            AGE
+redis-master     10.0.136.3       <none>            6379/TCP      app=redis,role=master,tier=backend  1h
+redis-slave      10.0.21.92       <none>            6379/TCP      app=redis,role=slave,tier=backend   1h
 ```
```

Comment:
Refer to https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/ui.md, in case they would like see the resources in the UI.